bundle/manifests/oadp.openshift.io_cloudstorages.yaml

```diff
@@ -96,5 +104,5 @@ status:
   acceptedNames:
     kind: ""
     plural: ""
-  conditions: []
-  storedVersions: []
+  conditions: null
+  storedVersions: null
```

Comment:
checking the changes done by the update of controller gen
- this empty list to null seems harmless
- description changes are harmless
- `x-kubernetes-map-type: atomic` were introduced by packages we use (example https://github.com/kubernetes/api/blob/v0.29.0/core/v1/types.go#L2272 ), and seem ok as well. Ref about these changes https://kubernetes.io/docs/reference/using-api/server-side-apply/#merge-strategy